docs/adr/2019-04-23-transcript-architecture.md

```diff
@@ -156,9 +160,10 @@ only IP address based authentication.
 ## Decision Outcome
```

Comment:
I am not clear from this ADR, which option are we going for, 1,2, or 3? or are we still deciding?

Reply:
so we are going for 3 because the first two are really old options :)
Trying to consolidate what we came up with in the 3rd option.